tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -424,6 +424,17 @@ public:
 // 3. The column is an RVec, the collection is not a vector
 // 4. The column is an RVec, the collection is a vector
 
+template <typename T, typename COLL>
+void FillColl(T& v, COLL c) {
+   c->emplace_back(v);
+}
+
+// Use push_back for bool since some compilers do not support emplace_back.
+template <typename COLL>
+void FillColl(bool v, COLL c) {
```

Comment:
`std::vector<bool>` doesn't have `emplace_back`, but other containers do

```suggestion
void FillColl(bool v, std::vector<bool> &c) {
```

Reply:
But now I should put a `inline` in front, right?

Reply:
Since it's not a template anymore.

Reply:
correct